Resources/ServerInfo/_NF/Guidebook/Rules/12_Security.xml

```diff
@@ -15,4 +15,8 @@ You must follow the law and be an example to the rest of the community when play
     -- Excess funds received from the sale of criminal assets are to be returned to their original owners.
     -- These assets are not to be plundered or embezzled by security forces.
     -- This is a strict zero-tolerance policy.
-</Document>
+    - The NFSD are allowed to use their knowledge of ship layouts in order to aid ship searches.
+    - NFSD who encounter a Syndicate dead drop are allowed to seize the contents of the pod and redeem it for FUCs at the NFSD Outpost.
+    -- NFSD are not allowed to activate dead drops themselves or knowingly work with persons who have activated one with the intentions of smuggling.
+    - Funds acquired on board vessels belonging to The Syndicate can be seized and deposited into the NFSD bank account. Officers may not keep these funds for personal use.
```

Comment:
I think rather than "funds" or "assets", the word "money" is preferable. The NFSD should not be hoovering up other valuable assets. "Money found ..." seems like the ideal wording to me.

Also, I'm not convinced by "non-player hostiles". If the NFSD help out on, say, a derelict where there just happens to be a pile of cash, I would say they should _not_ be permitted to take it. And then you get into rule lawyering about what counts as "hostiles". The Syndicate, yes. Wizards too? What about xenos, carps, dragons and other space critters? They sure ain't friendly. In reality, this rule is written _specifically_ because of the (considerable) piles of cash on the Syndicate unidentified vessel.

If one were to rephrase "The Syndicate", perhaps "hostile corporations" would be preferable.

Reply:
A mechanical reason to do this would be to trade in contraband items that appear for FUCs, or even help pay your force if it's larger than your stipend (which may be subject to change).  The rationale behind "non-player hostiles" is to avoid over-specificity and future rule rewrites if we one day add some other bluespace event.  As-is, this clause _should_ apply to LP Bravo, bluespace events (_currently_ the syndicate ship), and in a pinch, penal expeditions (N.B. a planetary facility isn't a vessel) - all PvE sources of loot.

Is Lord of War-style asset stripping the syndicate shuttle in bad taste?  Yeah.  Is taking a few contraband crates off of some intercepted smuggling ship in bad taste?  Probably not.  Is adding some sort of wizard event if/when #1522 happens reasonable?  I'd argue so.

Reply:
This clause is intended to refer specifically to loose cash on bluespace events. Contraband can always be seized by the NFSD and may _never_ be sold for spesos, only redeemed for FUCs. Contraband handling is covered by space law.

Reply:
Updated to be more specific and use the word 'money' instead. Resolving.